spyder/app/tests/test_mainwindow.py

```diff
@@ -266,6 +266,60 @@ def test_filter_numpy_warning(main_window, qtbot):
     CONF.set('variable_explorer', 'minmax', False)
 
 
+@pytest.mark.slow
+@flaky(max_runs=3)
+@pytest.mark.use_introspection
+@pytest.mark.skipif(PY2,
+                    reason="Times out in PY2")
+def test_get_help_combo(main_window, qtbot):
+    """
+    Test that Help works when called from the Editor and the IPython console.
```

Comment:
Please change this to

    Test that Help works when called from the IPython console.

Reply:
Sorry, that's not correct. This is:

    Test that Help combo box is working as expected.

Reply:
Since we have some extra characters, we should probably be clearer and more explicit about what ``as expected`` specifically means in this context.

Reply:
Ok, let's change this to

    Test that Help can get display docstrings for objects introduced on its combobox.

Reply:
Fixed some issues:

```
Test that Help can display docstrings for names typed in its combobox.
```